ocp4/profiles/moderate.profile

```diff
@@ -230,19 +231,19 @@ selections:
     - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    - package_fapolicyd_installed
-    - service_fapolicyd_enabled
+    #- package_fapolicyd_installed
+    #- service_fapolicyd_enabled
```

Comment:
This is a vertical and procurement requirement.

Reply:
Unfortunately, for now, these packages do not ship in CoreOS. Once the packages ship the rules can be re-enabled.

Reply:
eg every CoreOS box would be flagged for missing this package, but it is not available, so seems like unnecessary pain. Tracking of fapolicyd inclusion is done at the product level for now.